File: ModuleDaily/src/main/java/com/qihuan/dailymodule/model/ApiFactory.java
package com.qihuan.dailymodule.model;

import com.qihuan.commonmodule.net.ApiManager;

/**
 * ApiFactory
 *
 * @author qi
 * @date 2018/5/21
 */
public class ApiFactory {
    public static ZhihuApi getApi() {
        return ApiManager.getInstance().getApi(ZhihuApi.class);
    }
}

File: ModuleDaily/src/main/java/com/qihuan/dailymodule/model/ApiFactory.kt
package com.qihuan.dailymodule.model

import com.qihuan.commonmodule.net.ApiManager

/**
 * ApiFactory
 *
 * @author qi
 * @date 2018/5/21
 */
object ApiFactory {
    val api: ZhihuApi
        get() = ApiManager.instance.getApi(ZhihuApi::class.java)
}
